docs/user-guide/logging/sdl.md

```diff
@@ -1,14 +1,17 @@
 ---
 assignees:
-- lavalamp
-- satnam6502
+- crassirostris
+- piosz
 
 ---
 
-A Kubernetes cluster will typically be humming along running many system and application pods. How does the system administrator collect, manage and query the logs of the system pods? How does a user query the logs of their application which is composed of many pods which may be restarted or automatically generated by the Kubernetes system? These questions are addressed by the Kubernetes **cluster level logging** services.
+Before reading this page, it's recommended to take a look at [a general description](/docs/user-guide/logging/overview) of how logging works in kubernetes.
```

Comment:
"Before reading this page, it's recommended to familiarize yourself with the [overview of logging in Kubernetes](/docs/user-guide/logging/overview)."